pages/ugm2024.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: UGM 2024
+---
+
+## When, Where, and How
+The UGM will be held in a live, in-person format across 3 days, August 21-23, 2024, in London, UK ([Bush House](https://www.kcl.ac.uk/visit/bush-house) at King's College London).
```

Comment:
mention Thomas Young Centre!

Reply:
Added "in partnership with TYC" as suggested above.